packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -90,18 +92,37 @@ function onDidStartTaskProcessHandler(event: vscode.TaskProcessStartEvent): void
   }
 }
 
-function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): void {
+async function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): Promise<void> {
   const task = event.execution.task;
   const activeTerminal = vscode.window.activeTerminal;
 
   if (task.scope !== undefined && isTeamsfxTask(task)) {
     allRunningTeamsfxTasks.delete({ source: task.source, name: task.name, scope: task.scope });
   } else if (isNpmInstallTask(task)) {
     try {
-      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugNpmInstall, {
-        [TelemetryProperty.DebugNpmInstallName]: task.name,
-        [TelemetryProperty.DebugNpmInstallExitCode]: event.exitCode + "",
-      });
+      let npmInstallErrorMessage: string | undefined;
```

Comment:
Add try cache to the get-error-message part since we do not want to loss any npm install complete telemetry.

Reply:
Added.